push/service.go

```diff
@@ -55,17 +48,8 @@ func newClient(cert tls.Certificate) (*http.Client, error) {
 	return &http.Client{Transport: transport}, nil
 }
 
-// Push notification to APN service after performing serialization.
-func (s *Service) Push(deviceToken string, headers *Headers, payload interface{}) (string, error) {
```

Comment:
Removed the JSON marshalling version of Push and renamed PushBytes. It's awkward in the case of Queue because there are errors that can be returned before queueing and then other errors from the response. Plus using json.Marshal directly in examples makes it obvious that any JSON can be used and that one payload can be reused for multiple notifications.